frameworks/g-cloud-9/questions/services/securityGovernanceStandards.yml

```diff
@@ -15,8 +15,10 @@ type: checkboxes
 options:
   - label: CSA CCM v3.0
     value: csa_ccm
+    filter_label: CSA CCM version 3.0
```

Comment:
When discussed with Roz the preference was for version to be written in full.

Should we keep the filter_label or edit the label (something we haven't been doing so far)